README.md

```diff
@@ -1,6 +1,6 @@
 [![AD Test](https://github.com/opensearch-project/anomaly-detection/workflows/Build%20and%20Test%20Anomaly%20detection/badge.svg)](https://github.com/opensearch-project/anomaly-detection/actions?query=workflow%3A%22Build+and+Test+Anomaly+detection%22+branch%3A%22main%22)
 [![codecov](https://codecov.io/gh/opensearch-project/anomaly-detection/branch/main/graph/badge.svg?flag=plugin)](https://codecov.io/gh/opensearch-project/anomaly-detection)
-[![Documentation](https://img.shields.io/badge/doc-reference-blue)](https://opendistro.github.io/for-elasticsearch-docs/docs/ad/)
+[![Documentation](https://img.shields.io/badge/doc-reference-blue)](https://docs-beta.opensearch.org/monitoring-plugins/ad/index/)
```

Comment:
Does `lychee` support a deny list of regex, for example any link starting with  `https://opendistro.github.io` will be invalid.

Reply:
From [here](https://github.com/lycheeverse/lychee#commandline-parameters), don't believe so, although that could be helpful.

Reply:
Got it. I created an issue to ask deny list feature: https://github.com/lycheeverse/lychee/issues/267